src/lib/games.d.ts

```diff
@@ -1,21 +1,23 @@
-export enum category {
+export enum Category {
 	tik = 'tikspel',
 	bal = 'balspel',
 	loop = 'loopspel',
-	reactie = 'reactiespel'
+	reactie = 'reactiespel',
```

Comment:
Waar kunnen deze het beste capitalized worden? Is dat hier, in de API of in het component?

Reply:
Moeten even goed kijken inderdaad, want er zit ook hoofdletter gevoeligheid in de filters op de API. Daar kan ik natuurlijk ook nog een Regular Expression voor gebruiken

Reply:
Kunnen dit ook visueel capitalise doen via CSS

Reply:
Dat had ik inderdaad ook al gedaan, maar toen bleek dat die filters niet werkte, dus was ik er mee gestopt. Ik denk dat je dit soort dingen gewoon altijd in CSS moet doen. Gaat namelijk alleen om wat je visueel ziet.